mesonbuild/wrap/wrap.py

```diff
@@ -228,6 +242,40 @@ def get_hg(self, p):
                 subprocess.check_call(['hg', 'checkout', revno],
                                       cwd=checkoutdir)
 
+    def get_build_file(self, p):
+        build_file = p.get('build_file')
+        print('Looking for %s' % build_file)
+        checkoutdir = os.path.join(self.subdir_root, p.get('directory'))
+        bf_name = os.path.basename(build_file)
```

Comment:
__[flake8]__
* [F841] local variable 'bf_name' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->